### ISSUE TITLE
about batch normalization

### ISSUE THREAD

**Issue by yansd-c** [james34602/SpleeterRT#2]
Great works!
I read your code, but I am confused about the batch norm operation. I think batch norm in model prediction is (x-moving_mean)/moving_var * gamma + beta., while in the code is batchNorm[ _ + s] * val + batchNorm[ _ ]. I am not sure if it is correct.

**Comment by james34602** [comment]
Hello
The batch normalization equation was heavily modified for the model(Especially follow the way official Tensorflow Spleeter), which simplify computation.
